packages/grub/grub.spec

```diff
@@ -177,7 +177,8 @@ sed -e "s,__VERSION__,%{version},g" %{S:3} > sbat.csv
 popd
 
 %install
-MODS="configfile echo ext2 gptprio linux normal part_gpt reboot sleep zstd search"
+MODS=(configfile echo ext2 gptprio linux normal part_gpt reboot sleep zstd search)
+MODS+=(pgp crypto gcry_sha256 gcry_sha512 gcry_dsa gcry_rsa)
```

Comment:
> Include the crypto modules only if the Secure Boot feature flag is set?

I'd originally done this, but I've been trying to eliminate variant-specific package builds where possible, since they're all going to be awkward for out-of-tree builds.

Reply:
That's okay with me. I'll need to read up more on out-of-tree to avoid introducing similar foot guns myself.